test/apis/realtime/image-classifier-resnet50/cortex_inf.yaml

```diff
@@ -0,0 +1,22 @@
+- name: image-classifier-resnet50
+  kind: RealtimeAPI
+  pod:
+    port: 9000
+    max_concurrency: 8
+    containers:
+    - name: api
+      image: quay.io/cortexlabs-test/neuron-tf-serving:latest
```

Comment:
What do you think of adding this to the E2E test suite?

Reply:
Yes, I will create a separate ticket for this